<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouterDialog</class>
 <widget class="QDialog" name="RouterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Router Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edit_name">
       <property name="maxLength">
        <number>64</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_address">
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edit_address">
       <property name="maxLength">
        <number>64</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>User Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="edit_username">
       <property name="maxLength">
        <number>64</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="edit_password">
         <property name="maxLength">
          <number>64</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_show_password">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources/console.qrc">
           <normaloff>:/img/eye.png</normaloff>:/img/eye.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_comment">
     <property name="text">
      <string>Comment:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="edit_comment"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonbox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/console.qrc"/>
 </resources>
 <connections/>
</ui>
